tests/test_notifications.py

```diff
@@ -0,0 +1,953 @@
+from modularodm import Q
+from modularodm.exceptions import NoResultsFound
+import mock
+import unittest
+import datetime
+import collections
+import pytz
+from mako.lookup import Template
+from tests.base import OsfTestCase, capture_signals
+from nose.tools import *  # PEP8 asserts
+from framework.auth.signals import contributor_removed, node_deleted
+from framework.auth import Auth
+from website.util import web_url_for
+from website.notifications.model import Subscription, DigestNotification
+from website.notifications.constants import SUBSCRIPTIONS_AVAILABLE, NOTIFICATION_TYPES, USER_SUBSCRIPTIONS_AVAILABLE
```

Comment:
It would be better to use `from website.notifications import constants` and then just use `constants.USER_SU...`